generators/client/templates/angular/src/main/webapp/app/admin/health/_health.component.ts

```diff
@@ -44,6 +44,11 @@ export class <%=jhiPrefixCapitalized%>HealthCheckComponent implements OnInit {
         this.healthService.checkHealth().subscribe(health => {
             this.healthData = this.healthService.transformHealthData(health);
             this.updatingHealth = false;
+        }, error => {
+            if (error.status == 503) {
```

Comment:
this line produces a tslint warning:

WARNING in ./src/main/webapp/app/admin/health/health.component.ts
[48, 30]: == should be ===